shared/build.gradle.kts

```diff
@@ -101,6 +102,8 @@ kotlin {
         homepage = "https://github.com/touchlab/KaMPStarter"
         framework {
             isStatic = false
+            export(Deps.kermit)
+            transitiveExport = true
```

Comment:
Shouldn't need both of these. `transitiveExport = true` will export all dependencies so you wouldn't also need an explicit `export()` call. Though, pending file size investigation we might want to limit exports here.

Reply:
Re-reading the docs on transitiveExport, I don't think it's the case that it causes all dependencies to be exported. Instead it is a configuration of whether the dependencies of the libs you export will also be exported. In theory, we wouldn't need it, but there is a limitation called out in the docs which states that due to the metadata setup you need to either add this or explicitly export every platform